src/main/java/uk/nhs/hee/tis/trainee/sync/service/TcsSyncService.java

```diff
@@ -85,11 +90,23 @@ public class TcsSyncService implements SyncService {
 
   private final Map<String, Function<Record, TraineeDetailsDto>> tableNameToMappingFunction;
 
+  private final String deletePlacementEventTopicArn;
+  private final String deleteProgrammeMembershipEventTopicArn;
+
+  private final AmazonSNS snsClient;
+  private final ObjectMapper objectMapper;
+
   @Value("${service.trainee.url}")
   private String serviceUrl;
 
   TcsSyncService(RestTemplate restTemplate,
-      TraineeDetailsMapper mapper, PersonService personService) {
+                 TraineeDetailsMapper mapper,
+                 PersonService personService,
+                 @Value("${application.aws.sns.delete-placement-event}")
+                 String deletePlacementEventTopicArn,
+                 @Value("${application.aws.sns.delete-programme-membership-event}")
```

Comment:
I realise this is a bit clumsy, and would get considerably worse as we add more table queues - but it could be refactored at that point :)